api/validators/FeedbackControllerRequests.ts

```diff
@@ -1,27 +1,44 @@
 import { Type } from 'class-transformer';
-import { IsDefined, IsNotEmpty, MinLength, ValidateNested } from 'class-validator';
+import { Allow, IsDefined, IsNotEmpty, MinLength, ValidateNested } from 'class-validator';
 import { IsValidFeedbackType, IsValidFeedbackStatus } from '../decorators/Validators';
 import {
   SubmitEventFeedbackRequest as ISubmitEventFeedbackRequest,
   SubmitFeedbackRequest as ISubmitFeedbackRequest,
   UpdateFeedbackStatusRequest as IUpdateFeedbackStatusRequest,
+  FeedbackSearchOptions as IFeedbackSearchOptions,
   Feedback as IFeedback,
   FeedbackType,
   FeedbackStatus,
 } from '../../types';
+import { UserModel } from '../../models/UserModel';
+import { EventModel } from '../../models/EventModel';
 
 export class Feedback implements IFeedback {
   @IsDefined()
   @IsNotEmpty()
-  title: string;
+  user: UserModel;
 
   @IsDefined()
-  @MinLength(100)
+  @IsNotEmpty()
+  event: EventModel;
+
+  @IsDefined()
+  source: string;
+
+  @IsDefined()
```

Comment:
Same thing here. The purpose of the validator layer is to make sure the request object passing in has everything in the correct format. By including timestamp here, we're expecting there to be a "created at" timestamp before our feedback is processed by the controller/service and committed to the database. This is incorrect, since we'd only want a timestamp when the feedback is deemed appropriate to commit to the database. If you take a look at the model you created, we're defaulting the timestamp column to be the current time, which should take care of this field.